Classes/PhpBench/FrameworkEnabledBenchmark.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * (c) Contributors of the Neos Project - www.neos.io
+ * Please see the LICENSE file which was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Neos\BuildEssentials\PhpBench;
+
+use Neos\BuildEssentials\TestableFramework;
+use Neos\Flow\Core\Bootstrap;
+use Neos\Flow\Testing\RequestHandler\RuntimeSequenceInvokingRequestHandler;
+
+/**
+ * Tools for https://github.com/phpbench/phpbench
+ */
+class FrameworkEnabledBenchmark
+{
+    protected Bootstrap $flowBootstrap;
+
+    /**
+     * Use with phpbench BeforeClassMethods annotation to prepare persistence schema
+     */
+    public static function enablePersistence(): void
+    {
+        $_SERVER['FLOW_ROOTPATH'] = TestableFramework::calculateRootPath();
```

Comment:
```suggestion
        $this->withRootPath();
```

Reply:
or does a change here infer with `@BeforeMethods("withRootPath")`???

Reply:
should be fine! I just moved all this code so much around that I lost track of this easier way :D

Reply:
Wait no, this is static 😂 we can't  and it needs to be static for phpbench to run it as beforeClass hook.